gt.js

```diff
@@ -24,6 +26,12 @@ const __ = require('./_internal/placeholder')
  * Test if a left value is greater than (`>`) a right value. Either parameter may be an actual value.
  *
  * ```javascript [playground]
+ * const isAgeGreater21 = gt(21, 40)
+ *
+ * console.log(isAgeGreater21) // true
```

Comment:
this example will actually return false because 21 is not greater than 40

I would change this example to 

```
const age = 40

const isAgeGreaterThan21 = gt(age, 21)

console.log(isAgeGreaterThan21) // true
```